src/structures/Bot.ts

```diff
@@ -92,9 +93,12 @@ export class Bot extends EventEmitter {
             if(message.guild){
                 if(
                     (cmd.data.ownerOnly && (message.guild.ownerID !== message.author.id)) ||
-                    (cmd.data.permissions && !message.member.hasPermission(cmd.data.permissions))
+                    (cmd.data.memberPermissions && !message.member.hasPermission(cmd.data.memberPermissions))
                 ){
-                    return message.channel.send(this.messages.missingPermissions);
+                    return message.channel.send(this.messages.memberMissingPermissions);
+                }
+                if(cmd.data.botPermissions && !message.channel.permissionsFor(message.guild.me).has(cmd.data.botPermissions)){
+                   return message.channel.send(this.messages.clientMissingPermissions);
```

Comment:
It would be more consistent to use `bot` everywhere.
```suggestion
                   return message.channel.send(this.messages.botMissingPermissions);
```